### PR TITLE
Adjust SearchInDocument tool

### PR DESCRIPTION
**Pull request by beatrycze-volk**
Use plugin configuration and adjust template. It encapsulates part of changes from the PR #587.

This PR keeps everything working. No other PRs needed to be merged before it.

Preparation for #587

### CODE REVIEW

File: Resources/Public/Javascript/PageView/SearchInDocument.js [hunk 1]
```diff
@@ -56,11 +56,10 @@ $(document).ready(function() {
             "/",
             {
                 eID: "tx_dlf_search_in_document",
-                q: $( "input[name='tx_dlf[query]']" ).val(),
-                uid: $( "input[name='tx_dlf[id]']" ).val(),
-                start: $( "input[name='tx_dlf[start]']" ).val(),
-                encrypted: $( "input[name='tx_dlf[encrypted]']" ).val(),
-                hashed: $( "input[name='tx_dlf[hashed]']" ).val(),
+                q: $( "input[id='tx-dlf-search-in-document-query]']" ).val(),
```
Comment (albig) [inline Resources/Public/Javascript/PageView/SearchInDocument.js]:
```suggestion
                q: $( "input[id='tx-dlf-search-in-document-query']" ).val(),
```
This doesn't work. 

I think it's a search/replace mistake.

**Review by albig — CHANGES_REQUESTED** [review]

File: Resources/Public/Javascript/PageView/SearchInDocument.js [hunk 2]
```diff
@@ -15,9 +15,9 @@
   * @returns void
   */
 function nextResultPage() {
-    var currentstart = $("#tx-dlf-search-in-document-form input[name='tx_dlf[start]']").val();
+    var currentstart = $("#tx-dlf-search-in-document-form input[id='tx-dlf-search-in-document-start]']").val();
```
Comment (albig) [inline Resources/Public/Javascript/PageView/SearchInDocument.js]:
```suggestion
    var currentstart = $("#tx-dlf-search-in-document-form input[id='tx-dlf-search-in-document-start']").val();
```
All other lines are wrong, too. This is only a sample suggest.

**Review by albig — CHANGES_REQUESTED** [review]
Please review all lines. Not only the one I suggest a change.

**Comment by beatrycze-volk** [comment]
> Please review all lines. Not only the one I suggest a change.

Reviewed, I have changed all appearences of the error.

**Review by albig — APPROVED** [review]
This seems to work now. I'll merge it.

**Pull request merged**